recipes-connectivity/connman/connman/0001-main-add-ve-and-vb-to-the-default-interface-blacklis.patch

```diff
@@ -0,0 +1,32 @@
+From 85c6b7880473e72196203160c4057341c6cad20a Mon Sep 17 00:00:00 2001
+From: Krisztian Litkey <krisztian.litkey@intel.com>
+Date: Thu, 31 Mar 2016 15:14:51 +0300
+Subject: [PATCH 1/1] main: add ve- and vb- to the default interface blacklist.
+
+Add ve- and vb- to the list of default blacklisted interface
+prefixes. Such interfaces are usually used by systemd-nspawned
+containers and are typically managed by systemd-networkd.
+
+Upstream-Status: Pending
+
+Signed-off-by: Krisztian Litkey <krisztian.litkey@intel.com>
+---
+ src/main.c | 2 ++
+ 1 file changed, 2 insertions(+)
+
+diff --git a/src/main.c b/src/main.c
+index e46fa7b..f44a2ed 100644
+--- a/src/main.c
++++ b/src/main.c
+@@ -57,6 +57,8 @@ static char *default_blacklist[] = {
```

Comment:
not good, we're already providing an `/etc/connman/main.conf` with ostro-specific blacklist. Add it there.

actually sync with @pohly, @ipuustin and @mythi so we split that conf-file as per our stateless discussions. This way we can provide a default file and if people want a different setting for their product, they `.bbappend` that package and get a new setting without recompiling the whole connman. This file should be in `/usr` and if we ever need the one in `/etc/connman/main.conf` we can use a tool to manage it (ie: conffw), likely merging the value from `/usr` and whatever user wants.